Test/Tests/Network/IPFabric.Tests.ps1

```diff
@@ -47,6 +46,10 @@ Test-WithRetries 3 {
     Describe 'IP Fabric tests' -Tag Smoke, EnvSafe {
         Context "Gateway-less forwarding" {
             It 'Container can ping compute node in underlay network' {
+                $VHostName = $Testenv.Testbeds[1].GetVHostName()
+                $ComputeAddressInUnderlay = Invoke-Command -Session $Testenv.Testbeds[1].GetSession() -ScriptBlock {
```

Comment:
This probably should be in testbed class. If you'd add "todo" to change it in the future, I'd be grateful.

Reply:
```suggestion
                # TODO Move getting IP for interface to Testbed class
                $ComputeAddressInUnderlay = Invoke-Command -Session $Testenv.Testbeds[1].GetSession() -ScriptBlock {
```